### ISSUE TITLE
Do electron apps work?

### ISSUE THREAD

**Issue by ghost** [NodeOS/NodeOS#421]
Is it possible to create an electron app for NodeOS to act as an application or GUI?

**Comment by RossComputerGuy** [comment]
Yeah BlueLight does a sort of similar thing or will https://bluelightos.github.io/ 20.12.2018, 21:50, "Dilan Omer" <notifications@github.com>:Is it possible to create an electron app for NodeOS to act as an application or GUI?

**Comment by ghost** [comment]
But isn’t blue light a complete operating system, rather than something like Electron. I’ll check it out though. Thanks

**Comment by RossComputerGuy** [comment]
Well, in a future update I'll be testing chromium vs electron application running OS.js.20.12.2018, 21:54, "Dilan Omer" <notifications@github.com>:But isn’t blue light a complete operating system, rather than something like Electron. I’ll check it out though. Thanks

**Comment by ghost** [comment]
I might play around with Bluelight then, because I think it’ll suit my needs a little better than NodeOS, as it already has the browser embedded. Thanks

**Comment by RossComputerGuy** [comment]
At least wait until I release 0.1.5-prealpha because that adds an installer and office writer app.20.12.2018, 21:57, "Dilan Omer" <notifications@github.com>:I might play around with Bluelight then, because I think it’ll suit my needs a little better than NodeOS, as it already has the browser embedded. Thanks

**Comment by ghost** [comment]
Mmmm... I will probably play around with it a little bit for now and focus on NodeOS. Once you’ve got in an installer, I’ll them push focus onto Bluelight. You’ve got a new fan : )

**Comment by RossComputerGuy** [comment]
Thanks the installer is being shipped in the next update which is being shipped in less than an hour.20.12.2018, 22:02, "Dilan Omer" <notifications@github.com>:Mmmm... I will probably play around with it a little bit for now and focus on NodeOS. Once you’ve got in an installer, I’ll them push focus onto Bluelight. You’ve got a new fan : )

**Comment by piranna** [comment]
> Yeah BlueLight does a sort of similar thing or will https://bluelightos.github.io/

I approve that. Interested to see where your project ends :-)

**Comment by kwe26** [comment]
> > Yeah BlueLight does a sort of similar thing or will https://bluelightos.github.io/
> 
> I approve that. Interested to see where your project ends :-)

the project is disappered and its offical github website and youtube or discord
